packages/x-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.2.1",
+  "version": "6.6.0",
```

Comment:
I don't see any functional changes that were made to this module. What do you think about not releasing it? 🤔

Reply:
I think we can skip it, I left a comment about it here https://github.com/mui/mui-x/pull/9184#discussion_r1212759010